whois-update/src/main/java/net/ripe/db/whois/update/dns/zonemaster/domain/ZonemasterRequest.java

```diff
@@ -46,15 +46,24 @@ public enum Method {
         private String method;
         private int id;
 
+
         Method(final String method, final int id) {
             this.method = method;
             this.id = id;
         }
 
+        public static Method getObjectByMethod(String method){
+            for (ZonemasterRequest.Method object: Method.values()) {
+                if (method.equals(object.method)) {
```

Comment:
Is this added because of the test cases?

Reply:
removed